_studio/shared/include/mfx_config.h

```diff
@@ -126,10 +126,10 @@
 #define CMAPIUPDATE
 
 #define MFX_ENABLE_HEVCEHW_REFACTORING
- 
+
 #if defined(MFX_ENABLE_HEVCEHW_REFACTORING)
     #define MFX_ENABLE_HEVCEHW_REFACTORING_LIN_GEN9
     #define MFX_ENABLE_HEVCEHW_REFACTORING_LIN_GEN11
-#endif //defined(MFX_ENABLE_HEVCEHW_REFACTORING) 
+#endif //defined(MFX_ENABLE_HEVCEHW_REFACTORING)
```

Comment:
can you, please, drop the comment if you will push any changes to this PR? It's clearly seen to which if belongs endif...